Add supported versions & reorder check startup

milohuman opened the pull request:
Added 1712 & later as a supported version. Also reordered the "Get the startup status for Azure Stack" section to be after the "Start Azure Stack" sections. Doesn't make sense to explain how to check the startup status before the reader has learned how to start azure stack.

azurecla commented:
Hi __@alexwun__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (alexwun). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

covetocove committed:
Add supported versions & reorder check startup

Added 1712 & later as a supported version. Also reordered the "Get the startup status for Azure Stack" section to be after the "Start Azure Stack" sections. Doesn't make sense to explain how to check the startup status before the reader has learned how to start azure stack.

v-shils committed:
Merge branch 'master' into patch-2
